pkg/apis/componentconfig/types.go

```diff
@@ -56,4 +56,9 @@ type ControllerManagerConfiguration struct {
 	// ResyncInterval is the interval on which the controller should re-sync
 	// all informers.
 	ResyncInterval time.Duration
+
+	// ConcurrentSyncs is the number of service catalog resources that are
```

Comment:
This is the number of workers for each resource. As far as I understand it, the "number of service catalog resources " is `concurrent syncs * #of resource types`. Here, by default, 5 * 4 = 20.